client/state/purchases/selectors/should-revert-atomic-site-before-deactivation.js

```diff
@@ -19,10 +20,15 @@ export const shouldRevertAtomicSiteBeforeDeactivation = ( state, purchaseId ) =>
 
 	const purchase = getByPurchaseId( state, purchaseId );
 
+	const isAtomicSupported = ( productSlug ) =>
```

Comment:
Can we still use `isAtomicSupportedProduct` which is more descriptive instead of `isAtomicSupported`?

Reply:
I've reverted the name change 👍